code/modules/antagonists/blob/blob/theblob.dm

```diff
@@ -21,14 +21,15 @@
 	var/mob/camera/blob/overmind
 
 /obj/structure/blob/Initialize(mapload, owner_overmind)
-	overmind = owner_overmind
-	var/area/Ablob = get_area(loc)
-	if(Ablob.blob_allowed) //Is this area allowed for winning as blob?
-		overmind.blobs_legit += src
+	. = ..()
+	if(owner_overmind)
+		overmind = owner_overmind
+		var/area/Ablob = get_area(loc)
```

Comment:
get_area(src)

Reply:
I want to find who made this original blob code and hit them with a tree branch.